src/test/java/controllers/ApiControllerDocTesterTest.java

```diff
@@ -39,16 +40,21 @@ public void testGetIndex() {
                         testServerUrl().path(URL_INDEX)));
 
         assertThat(response.payload, containsString("Choose A Deck"));
-        //assertThat(response.payload, containsString("BAM!"));
     }
 
-    //@Test
-    //public void testAcesUp() {
-    //    Response response = makeRequest(
-    //            Request.GET().url(
-    //                    testServerUrl().path(URL_ACES_UP)));
-    //    assertThat(response.payload, containsString("Welcome to Ace's Up"));
-        //assertThat(response.payload, containsString("columnOfCards"));
-    //}
+    @Test
+    public void testAcesUp() {
+        Response response = makeRequest(
+                Request.GET().url(
+                        testServerUrl().path(URL_ACES_UP)));
+        assertThat(response.payload, containsString("Welcome to Ace's Up"));
+    }
+    @Test
+    public void testSPAcesUp() {
+        Response response = makeRequest(
+                Request.GET().url(
+                        testServerUrl().path(URL_SP_ACES_UP)));
+        assertThat(response.payload, containsString("Welcome to Ace's Up"));
```

Comment:
Would it be difficult to change the message to say "...Ace's Up (Spanish)" or something along those lines?

May make it easier to distinguish which version of the game is being played.